### ISSUE TITLE
allow --assignee to unassign everyone

### ISSUE THREAD

**Issue by maxice8** [profclems/glab#483]
**Describe the feature or problem you'd like to solve**

Allow --assignee to unassign all users

**Propose a Solution**

add the special value `@nobody` which is translated to an user with `id = 0`, the value 0 is used in the gitlab API tounassign all users

**Additional context**

Example:

```console
foo@bar:~$ glab mr view 1
<mr assigneed to profclems>
foo@bar:~$ glab mr update --assignee="@nobody"
[trucated]
foo@bar:~$ glab mr view 1
<mr without assignees>
```

**Comment by profclems** [comment]
+1 for the @nobody suggestion. We can also allow users to specify 0.

Both `--assignee @nobody` and `--assignee 0` can be accepted imho

**Comment by maxice8** [comment]
Will be fixed in #513, it adds support for using `--assign @nobody` and `--unassign`
